src/sidebar/components/NotebookResultCount.js

```diff
@@ -1,9 +1,19 @@
+import propTypes from 'prop-types';
+
 import useRootThread from './hooks/use-root-thread';
-import { useStoreProxy } from '../store/use-store';
 import { countVisible } from '../helpers/thread';
 
 import Spinner from './Spinner';
 
+/**
+ * @typedef NotebookResultCountProps
```

Comment:
`NotebookResultCount` takes several more props as of these changes, most of which are unrelated to the changes at hand but an attempt to make component "sophistication" balanced. In this case, the removal of child component dependencies on `storeProxy`.